docs/release-notes/8.3.asciidoc

```diff
@@ -1,10 +1,27 @@
 [[release-notes-header-8.3.0]]
 == 8.3
 
+[discrete]
+[[release-notes-8.3.3]]
+=== 8.3.3
+
+[discrete]
+[[bug-fixes-8.3.3]]
+==== Bug fixes and enhancements
+* Fixes a bug that prevented the *Create field* button from appearing in the Fields browser when you accessed it from a Timeline created using the Alerts page's *Open in timeline* button ({pull}135842[#135842]).
+* Removes the unsupported `matches` operator from the *Add Rule Exception* flyout ({pull}136340[#136340]).
+* Prevents rule execution log events from being wrongly ordered when the maximum number of events are reached and events are filtered by status ({pull}131675[#131675]).
+* Isolates the query used for the *First seen* and *Last seen* fields on the Users, Hosts, and Network details pages to improve page load times and UI performance ({pull}133539[#133539]).
```

Comment:
Verified with @jamster10 that this PR needs to be removed from the 8.3.3 release notes. It will need to be doc'd as a fixed bug in the [8.4.0 release notes](https://github.com/elastic/security-docs/issues/2175) though.